root/views.py

```diff
@@ -54,6 +54,97 @@ def about(request):
 def sponsors(request):
     return render(request, 'sponsors.html', context={'page': 'sponsors'})
 
+
 def travelling(request):
     # Info taken from: https://www.mapsofindia.com/pune/travel-to-pune.html
     return render(request, 'travelling.html', context={'page': 'travelling'})
+
+
+def schedule(request):
+    context = [
```

Comment:
Store the dict separately in some json file and read from that file here so that to add schedule only json file needs to be changed without messing up the python file.